src/main/osd/osd_elements.c

```diff
@@ -1082,12 +1086,12 @@ static void osdElementEfficiency(osdElementParms_t *element)
 {
     int efficiency = 0;
     if (sensors(SENSOR_GPS) && ARMING_FLAG(ARMED) && STATE(GPS_FIX) && gpsSol.groundSpeed >= EFFICIENCY_MINIMUM_SPEED_CM_S) {
-        const int speedX100 = osdGetSpeedToSelectedUnit(gpsSol.groundSpeed * 100); // speed * 100 for improved resolution at slow speeds
-        
-        if (speedX100 > 0) {
-            const int mAmperage = getAmperage() * 10; // Current in mA
-            efficiency = mAmperage * 100 / speedX100; // mAmperage * 100 to cancel out speed * 100 from above
+        const float speed = (float)osdGetSpeedToSelectedUnit(gpsSol.groundSpeed);
+        if (speed > 0.0f) {
```

Comment:
Wait a sec... 

`if (gpsSol.groundSpeed >= EFFICIENCY_MINIMUM_SPEED_CM_S)`

is already tested in line 1088, doesn't that mean that `float speed` is already guaranteed to be greater than `0.0f` anyways? In other words: can we remove this conditional in line 1090?

Reply:
Yes it does appear to be redundant. I was modeling it after the original code which Checked to be sure speedX100 was greater than zero. I'll remove the extra condition. Do I need to squash the two commits?

Reply:
Yeah please. But you can also do
```
git reset --soft HEAD~1
```
then remove the conditional and then
```
git add -a
git commit -m "Update for Battery Efficiency Fix Issue #11277"
git push --force
```

Reply:
Or 
```
git commit -a --amend --no-edit
git push origin +OSDBattEfficiencyFix
```